tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_private_navigation.py

```diff
@@ -16,12 +17,20 @@ def setUp(self):
         self.test_url = self.marionette.absolute_url('mozilla.html')
 
     def test_browser_private_navigation(self):
-        search = Search(self.marionette)
-        search.launch()
-        private_window = search.open_new_private_window()
+        self.device.hold_home_button()
+
+        cards_view = CardsView(self.marionette)
+        private_window = cards_view.open_new_private_window()
+
         browser = private_window.go_to_url(self.test_url)
         browser.switch_to_content()
         Wait(self.marionette).until(lambda m: m.title == 'Mozilla')
 
+        self.device.hold_home_button()
+
+        cards_view = CardsView(self.marionette)
+        cards_view.wait_for_cards_view()
+        search = cards_view.open_new_browser()
+        # This is to switch the marionette context to the browser content
```

Comment:
Yes, but that can't work currently, because the locators for the browser app are far too generic, they pick up the first browser instance that's out there.
We would need the fix in https://bugzilla.mozilla.org/show_bug.cgi?id=1205262 for that.
Is it ok to just leave it like this for now and as soon as I have https://bugzilla.mozilla.org/show_bug.cgi?id=1205262 fixed to make this work as you suggested?